edge-hub/docker/linux/arm32v7/Dockerfile

```diff
@@ -15,5 +15,5 @@ EXPOSE 443/tcp
 USER edgehubuser
 ENV OptimizeForPerformance false
 ENV MqttEventsProcessorThreadCount 1
-CMD echo "$(date --utc +"[%Y-%m-%d %H:%M:%S %:z]"): Starting Edge Hub" && \
+CMD echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Starting Edge Hub" && \
```

Comment:
nice to have: inject this time format by ARG.  May be just created a pbi to follow up later.

Reply:
Seems a bit of an overkill.. what would be the scenario?

Reply:
nice-to-have, just for code optimization.